taxi/urls.py

```diff
@@ -7,6 +7,11 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
+    CarCreateView,
+    CarUpdateView, CarDeleteView
```

Comment:
Move `CarDeleteView` to a new line so that all imports will be in the same style.